index.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
```

Comment:
Let's move this file inside `src` directory. This way, all our actual code will be inside `src/` directory.